app/assets/javascripts/transactions.js

```diff
@@ -26,6 +26,15 @@ $(function () {
     e.stopPropagation();
   });
 
+  $(document).on('click', '#clear', function(e) {
```

Comment:
@Gataniel let's use some class name, like `clear-form`, so we will able to use it in a few forms on one page in future.